msgcntr/messageforums-api/src/bundle/org/sakaiproject/api/app/messagecenter/bundle/Messages.properties

```diff
@@ -213,6 +216,9 @@ cdfm_moderate_forum=Moderate topics in forum
 cdfm_moderate_forums=Moderate topics in forums
 cdfm_postFirst=Require users to post before reading
 cdfm_postFirst_warning=You must post to this topic before you can view any messages
+cdfm_postAnonymous=Posts are anonymous
+cdfm_revealIDsToRoles=Reveal identities to roles with the "Identify Anonymous Authors" permission below
```

Comment:
Ideally, but the string would have to be assembled in java code somewhere (it's referenced in a jsp). Probably simpler to just leave it typed out in the message bundle.